models/articles.models.js

```diff
@@ -39,4 +40,27 @@ exports.selectArticles = (sort_by) => {
              });
      };
 
-    
\ No newline at end of file
+exports.updateArticleByArticleId =(article_id, inc_votes)=>{
+     
+      return db.query(`SELECT votes FROM articles WHERE article_id =$1`,[article_id])
```

Comment:
no need to query db for current amount of article votes.  You can just set them in the sql string (SET votes = votes + $1)